model/page.rb

```diff
@@ -22,7 +22,7 @@ def pagerduty_client
   def trigger
     return unless Config.pagerduty_key
 
-    incident = pagerduty_client.incident(Digest::MD5.hexdigest(id))
+    incident = pagerduty_client.incident(Digest::MD5.hexdigest(tag))
```

Comment:
I'd suggest changing this to HMAC, if at least for no other reason is that any use of MD5 will generate a footnote in every security audit we undertake.

You don't need to use a secret key to use HMAC in a practical way, you could even have it be a string like "clover page tag".  if you dig into the [SCRAM specification](https://www.rfc-editor.org/rfc/rfc5802), you can see how they use it, with literal uses of "Client Key" and "Server Key" as key input to HMAC:

```

     SaltedPassword  := Hi(Normalize(password), salt, i)
     ClientKey       := HMAC(SaltedPassword, "Client Key")
     StoredKey       := H(ClientKey)
     AuthMessage     := client-first-message-bare + "," +
                        server-first-message + "," +
                        client-final-message-without-proof
     ClientSignature := HMAC(StoredKey, AuthMessage)
     ClientProof     := ClientKey XOR ClientSignature
     ServerKey       := HMAC(SaltedPassword, "Server Key")
     ServerSignature := HMAC(ServerKey, AuthMessage)
```